tests/ui/arch/trace_ray_khr.rs

```diff
@@ -2,7 +2,7 @@
 
 #[spirv(ray_generation_khr)]
 pub fn main(
-    acceleration_structure: &spirv_std::ray_tracing::AccelerationStructureKhr,
+    #[spirv(uniform, descriptor_set = 0, binding = 0)] acceleration_structure: &spirv_std::ray_tracing::AccelerationStructureKhr,
```

Comment:
wasn't uniform in this position only for uniform buffers?

Reply:
Yep, I was just rushing to fix it and didn't type `_constant`, it should be `uniform_constant` but that shouldn't be needed once #567 is merged.